http.c

```
@@ -4910,6 +4993,18 @@ parse_authority(struct evhttp_uri *uri, char *s, char *eos, unsigned *flags)
 	} else {
 		cp = s;
 	}
+
+#ifndef _WIN32
+	if (*flags & EVHTTP_URI_UNIX_SOCKET && !strncmp(cp, "unix:", 5)) {
+		char *e = strchr(cp + 5, ':');
+		if (e) {
+			*e = '\0';
+			uri->unixsocket = mm_strdup(cp + 5);
+			return 0;
+		}
```

Let's return an error